attachment_edit/static/src/css/attachment_edit.css

```diff
@@ -0,0 +1,22 @@
+.openerp .oe_sidebar .oe_dropdown_menu li .oe-sidebar-attachment-edit
+{
+    position: absolute;
```

Comment:
There's no other option that to put an absolute position? Can you insert the button in the flow via a JQuery command?